g2core/board/ArduinoDue/gShield-pinout.h

```diff
@@ -258,12 +258,12 @@ pin_number kADC14_PinNumber = -1;  // Not physially pinned out
 
 
 // GRBL / gShield compatibility pins -- Due board ONLY
-
+// NOTE: These dont appear to work
```

Comment:
`INPUT_ACTION_CYCLE_START` is interesting. We automatically cycle-start, so we'd have to find a way to make that non-automatic. I guess for now that can be removed and just treated as a plain input (if it's not already, it may be). It would be easy to fake the wait-for-button-push-to-start by making the first line in the gcode something like `M101 ({in7:t})` (replacing 7 with whatever input it is).

`INPUT_ACTION_RESET` is also interesting. I don't recall the intention for that. I presume it's not board reset, there's already one of those, so maybe it's supposed to mean reset as in `M30`? @aldenhart would know.

`INPUT_ACTION_HOLD` should be identical to sending a `!` with the important exception that Chili (or whatever UI) didn't see it, and if they're not watching `stat` to see if we went into hold, they might not offer to resume. If the UI allows it, a `~` should do the trick, just like after a `!`. (AFAIK, this has not been tested, and if it doesn't work then please make an issue.)

Again, thank you!